curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61695c4aad56f95497c19583.md

```diff
@@ -7,7 +7,10 @@ dashedName: step-3
 
 # --description--
 
-Remember that the `title` element gives search engines extra information about the page. It also tells browsers what text to display in the title bar when the page is open, and on the tab for the page.
+Remember that the `title` element gives search engines extra information about the page. Apart from that, it displays the content of that `title` element in two more ways:
```

Comment:
You can remove "apart from that" and say "It also displays the content of the `title` element in two other ways":

Reply:
Done